mobile/native/src/referrals.rs

```diff
@@ -0,0 +1,18 @@
+use crate::commons::reqwest_client;
+use crate::config;
+use anyhow::Result;
+use commons::ReferralStatus;
+use reqwest::Url;
+
+pub(crate) async fn get_referral_status() -> Result<ReferralStatus> {
```

Comment:
🔧 I believe this is not used anymore. The endpoint has also been moved to the admin api.

I would remove that function.